packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -167,19 +171,151 @@ describe("Hotkeys", () => {
             expect(testCombo).to.equal(combo);
         });
 
-        function assertInputAllowsKeys(type: string, allowsKeys: boolean) {
-            comp = mount(<TestComponent />, { attachTo });
+        function runHotkeySuiteForKeyEvent(eventName: "keydown" | "keyup") {
+            it("triggers local and global hotkeys on keydown", () => {
```

Comment:
did not expect to see hardcoded "keydown" and "keyup" in test names here. should be parameterized?